addons/mass_mailing/views/mass_mailing.xml

```diff
@@ -281,10 +281,10 @@
                             string="Test Mailing"/>
                         <button name="send_mail" type="object"
                             attrs="{'invisible': ['|', ('state', '=', 'done'), ('body_html', '=', False)]}"
-                            class="oe_highlight" string="Send to All"/>
+                            class="oe_highlight" string="Send to All" confirm="Are you sure?"/>
```

Comment:
I would better say : "This will send the email to all recipients. Do you still want to proceed ?"